extract_timeseries_tar.py

```diff
@@ -152,17 +248,25 @@ def get_parser():
             # Note from AB
             # if multiple run, use run 2
             # if multiple session, use ses 1
-            fmri = layout.get(return_type='type', subject=subject, space='MNI152NLin2009cAsym',
+            fmri = layout.get(return_type='type', subject=subject, space=template,
                               desc='preproc', suffix='bold', extension='nii.gz')
+            brain_mask = layout.get(return_type='type', subject=subject, space=template,
+                                    desc='brain', suffix='mask', extension='nii.gz')
+            # TODO: check if brain_mask and fmri always come in pairs
+            # according to doc, desc-preproc_bold and desc-brain_mask should come in pairs
             for ii in range(len(fmri)):
                 file_entitiles = fmri[ii].entities
                 timeseries_root_dir = create_timeseries_root_dir(
                     file_entitiles, output_dir)
                 for dimension in ATLAS_METADATA[atlas_name]['dimensions']:
                     for resolution in ATLAS_METADATA[atlas_name]['resolutions']:
-                        print(f"\t dim{dimension} - res0{resolution}")
-                        masker, labels = create_atlas_masker(
-                            atlas_name, atlas_path, dimension, resolution)
+                        print(f"\t dim{dimension} - {resolution}")
+                        description_keywords = {'dimension': dimension,
```

Comment:
This is here where you can create a CURR_ATLAS object (comment from l 39-40).
Or MUCH EASIER make `generate_templateflow_parameters()` take arguments (resolution, components, name etc..) instead of the weird static/dynamic cur_atlas_meta object and even weirder `description_keywords` (I don't see the point of this dict).

Reply:
The reason of using a dictionary instead of just having dimension and resolution is because not all atlas will share the same type of entities. I want to have some capacity for adding new atlas in the the future. TLDR should have written tests.

Reply:
The only fixed entity in templateflow patter is quite limited, and some of the things you are thinking about, such as component, is defined by user under the description entity.